src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/target.py

```diff
@@ -94,6 +99,11 @@ def __init__(
         )
         self.target_structure = target_structure
         self.organizations = organizations
+        parameter_store = ParameterStore(DEPLOYMENT_ACCOUNT_REGION, boto3)
+        self.allow_empty_deployment_maps = parameter_store.fetch_parameter(
```

Comment:
if you change this to 
```suggestion
        self.allow_empty_deployment_maps = parameter_store.fetch_parameter("/adf/deployment-maps/allow-empty-target") == "TRUE"
```
then future reference to it will be a lot nicer because it's a bool.